eZ/Publish/Core/Persistence/Legacy/Content/UrlAlias/Gateway/DoctrineDatabase.php

```diff
@@ -877,7 +877,13 @@ public function loadPathData($id)
             if (empty($rows)) {
                 // Normally this should never happen
```

Comment:
hmm.

Reply:
This is here from before.

It happens when database is inconsistent, which is what @alongosz is working on fixing in improved url-alias re-generation command. 


That said, we can update the comment and the exception message to indicate way to resolve the issue is by enabling migration bundle and running `ezplatform:regenerate:legacy_storage_url_aliase`

Until @alongosz has finished his work history is lost, but hopefully we'll release the two changes together.

Reply:
Improved code in #2388 via c9945ce.